runtime/src/main/java/org/corfudb/runtime/RoutingQueueSenderClient.java

```diff
@@ -204,6 +206,10 @@ public void onNext(CorfuStreamEntries results) {
             }
         }
 
+        public Map<String, LRFullStateReplicationContext> getPendingFullSyncs() {
```

Comment:
can be default or protected.  Or you can add a getter on the pending full syncs map